mmseg/models/segmentors/base.py

```diff
@@ -104,6 +104,15 @@ def forward(self, img, img_metas, return_loss=True, **kwargs):
         should be double nested (i.e.  List[Tensor], List[List[dict]]), with
         the outer list indicating test time augmentations.
         """
+        if torch.onnx.is_in_onnx_export() or kwargs.pop('do_norm', False):
+            assert len(img_metas) == 1
+            img_norm_cfg = img_metas[0][0].get('img_norm_cfg', None)
+            if img_norm_cfg and img_norm_cfg['normalize_in_graph']:
+                mean = torch.tensor(img_norm_cfg['mean'])[None, ..., None,
+                                                          None]
+                std = torch.tensor(img_norm_cfg['std'])[None, ..., None, None]
+                img[0] = (img[0] - mean) / std
```

Comment:
L114
img[0] = (img[0] - mean) / std should be img = (img - mean) / std?

Reply:
If yes, it is better to move to the corresponding part.

Reply:
You're right that line should deal with each image in the list. Also it does happen to only be used when `return_loss=False` for now because it's only used in the onnx exporting script. I'll make both changes